src/app/items/containers/item-extra-time/item-extra-time-input/item-extra-time-input.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, effect, input, model, output } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { InputNumberModule } from 'primeng/inputnumber';
+import { ButtonIconComponent } from 'src/app/ui-components/button-icon/button-icon.component';
+
+@Component({
+  selector: 'alg-item-extra-time-input',
+  templateUrl: './item-extra-time-input.component.html',
+  styleUrls: [ './item-extra-time-input.component.scss' ],
+  standalone: true,
+  imports: [
+    FormsModule,
+    InputNumberModule,
+    ButtonIconComponent,
+  ],
+})
+export class ItemExtraTimeInputComponent {
+  initialValue = input.required<number>();
+  value = model<number | null>(null); // The p-inputNumber value is nullable
```

Comment:
In that option, I think the point was that value was no longer an input... so no longer a model but just a signal.

Reply:
I have just tested it that works.
```suggestion
  value = signal<number | null>(null); // The p-inputNumber value is nullable
```